### PR TITLE
[ocrvs-2169] Change user-profile office label

### PR DESCRIPTION
**Pull request by Kayumuzzaman**
Renamed `Assigned office` to `Assigned registration office` and fixed padding for user profile.

### CODE REVIEW

**Review by youshamahmood96 — APPROVED** [review]

File: packages/client/src/views/SysAdmin/Team/user/userProfilie/UserProfile.tsx [hunk 1]
```diff
@@ -387,7 +387,7 @@ class UserProfileComponent extends React.Component<Props, State> {
                       </InformationValue>
                     </InformationHolder>
                     <InformationHolder>
-                      <InformationTitle paddingRight={113}>
+                      <InformationTitle paddingRight={208}>
```
Comment (mushrafulhoque-dsi) [inline packages/client/src/views/SysAdmin/Team/user/userProfilie/UserProfile.tsx:390]: Why do we need this change ? Can you upload a screen shot for this ?

File: packages/client/src/i18n/messages/views/userSetup.ts [hunk 2]
```diff
@@ -170,7 +170,7 @@ const messagesToDefine: IUserSetupMessages = {
     id: 'userSetup.waiting'
   },
   assignedOffice: {
-    defaultMessage: 'Assigned office',
+    defaultMessage: 'Assigned registration office',
```
Comment (mushrafulhoque-dsi) [inline packages/client/src/i18n/messages/views/userSetup.ts:173]: I don't see any corresponding PR on niue repo for this. Is it coming ?

**Review by mushrafulhoque-dsi — CHANGES_REQUESTED** [review]

File: packages/client/src/i18n/messages/views/userSetup.ts (hunk 2 above)
Comment (Kayumuzzaman) [reply packages/client/src/i18n/messages/views/userSetup.ts:173]: I gave a PR on niue which is merged by mr. @youshamahmood96

File: packages/client/src/views/SysAdmin/Team/user/userProfilie/UserProfile.tsx (hunk 1 above)
Comment (Kayumuzzaman) [reply packages/client/src/views/SysAdmin/Team/user/userProfilie/UserProfile.tsx:390]: After padding: 
ocrvs-2169

**Review by mushrafulhoque-dsi — APPROVED** [review]

**Pull request merged**